youtube_dl/extractor/cultureunplugged.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+
+
+class CultureUnpluggedIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?cultureunplugged\.com/documentary/watch-online/play/(?P<id>\d+)/(?P<display_id>[^/]+)'
```

Comment:
You didn't make `display_id` part option. http://www.cultureunplugged.com/documentary/watch-online/play/53662 URLs works just fine in browser.